src/shiv/cli.py

```diff
@@ -125,6 +126,16 @@ def main(
     interpreter = validate_interpreter(python)
 
     with TemporaryDirectory() as working_path:
+        cwd = Path.cwd()
+        # distutils doesn't support using --target if there's a config file
+        # specifying --prefix. Homebrew's Pythons include a distutils.cfg that
+        # breaks `pip install --target` with any non-wheel packages. We can
+        # work around that by creating a setup.cfg specifying an empty prefix
+        # in the directory we run `pip install` from.
+        with Path(working_path, "setup.cfg").open('w') as f:
```

Comment:
maybe a better place for this would be in `clean_pip_env()` 🤔

Reply:
that’s exactly what i was going to suggest! makes perfect sense there